java/drivers/driver-hazelcast4plus/src/main/java/com/hazelcast/simulator/tests/map/LongByteArrayMapTest.java

```diff
@@ -141,8 +152,18 @@ public void pipelinedGet(final ThreadState state, @StartNanos final long startNa
     public class ThreadState extends BaseThreadState {
         private Pipelining<byte[]> pipeline;
         private int i;
+        private int fixedKeysDomain;
+
+        public ThreadState() {
+            if (fixedKeysPercentage > 0) {
+                fixedKeysDomain = keyDomain * (fixedKeysPercentage / 100);
```

Comment:
Since all variables involved are int, this results in integer division, which truncates decimals, e.g.:
25 / 100 = 0

We should probably use something like: `fixedKeysDomain = keyDomain * (fixedKeysPercentage / 100.0);`